packages/front-end/pages/settings/index.tsx

```diff
@@ -110,7 +110,15 @@ const GeneralSettingsPage = (): React.ReactElement => {
       displayCurrency,
       secureAttributeSalt: "",
       killswitchConfirmation: false,
-      requireReviews: false,
+      requireReviews: [
+        {
+          requireReview: false,
+          resetReviewOnChange: false,
+          environments: [],
+          projects: [],
+          tags: [],
+        },
+      ],
```

Comment:
Should this default to an empty array instead?

Reply:
we could but that would make it more confusing on the form as this would ensure that we have all the default values set.